### PR TITLE
4-quicksort.MD

### PR DESCRIPTION
**Pull request by vimalvijenthiran**
This is a PR for quick sort issue in python

### CODE REVIEW

**Review by Abishethvarman — APPROVED** [review]
Approved to merge

File: 1-data-structures/3-python/4-quicksort.MD [hunk 1]
```diff
@@ -0,0 +1,15 @@
+QuickSort is a popular sorting algorithm that follows the divide-and-conquer paradigm. It works by selecting a 'pivot' element from the array and partitioning the other elements into two sub-arrays according to whether they are less than or greater than the pivot. The sub-arrays are then recursively sorted.
```
Comment (Thevapriyan07) [inline 1-data-structures/3-python/4-quicksort.MD:1]: make this readme file with #Definition and add some other thing which are  mentioned in the issue
using proper way to write readme file

Comment (Abishethvarman) [inline 1-data-structures/3-python/4-quicksort.MD:1]: Add heading

**Review by Abishethvarman — CHANGES_REQUESTED** [review]

**Comment by Abishethvarman** [comment]
closed due to file relocation issue

**Pull request closed without merging**